docs/diginstroom/sip/1.0/usecases/gigapixel-artwork.md

```diff
@@ -0,0 +1,540 @@
+        <premis:linkingObjectIdentifier>
+            <premis:linkingObjectIdentifierType>UUID</premis:linkingObjectIdentifierType>
+            <premis:linkingObjectIdentifierValue>uuid-51F2C0F1-CA06-4B59-9605-54F7C91BA53F</premis:linkingObjectIdentifierValue>
+            <premis:linkingObjectRole valueURI="http://id.loc.gov/vocabulary/preservation/eventRelatedObjectRole/out">outcome</premis:linkingObjectRole>
+        </premis:linkingObjectIdentifier>
+        <premis:linkingObjectIdentifier>
+            <premis:linkingObjectIdentifierType>UUID</premis:linkingObjectIdentifierType>
+            <premis:linkingObjectIdentifierValue>uuid-187DA428-6BA1-4EB7-B786-CD4AF85A02B1</premis:linkingObjectIdentifierValue>
+            <premis:linkingObjectRole valueURI="http://id.loc.gov/vocabulary/preservation/eventRelatedObjectRole/out">outcome</premis:linkingObjectRole>
+        </premis:linkingObjectIdentifier>
+        <premis:linkingObjectIdentifier>
+            <premis:linkingObjectIdentifierType>UUID</premis:linkingObjectIdentifierType>
+            <premis:linkingObjectIdentifierValue>uuid-70dd6ae0-37fe-11ed-9b20-7e92631d7d28</premis:linkingObjectIdentifierValue>
+            <premis:linkingObjectRole valueURI="http://id.loc.gov/vocabulary/preservation/eventRelatedObjectRole/out">outcome</premis:linkingObjectRole>
+        </premis:linkingObjectIdentifier>
+        <premis:linkingObjectIdentifier>
+            <premis:linkingObjectIdentifierType>UUID</premis:linkingObjectIdentifierType>
+            <premis:linkingObjectIdentifierValue>uuid-4e475706-2752-4f77-9069-1f71c0e22572</premis:linkingObjectIdentifierValue>
+            <premis:linkingObjectRole valueURI="http://id.loc.gov/vocabulary/preservation/eventRelatedObjectRole/out">outcome</premis:linkingObjectRole>
+        </premis:linkingObjectIdentifier>
+        <premis:linkingObjectIdentifier>
+            <premis:linkingObjectIdentifierType>UUID</premis:linkingObjectIdentifierType>
+            <premis:linkingObjectIdentifierValue>uuid-ddcf9e36-35ae-11ed-b866-7e92631d7d27</premis:linkingObjectIdentifierValue>
+            <premis:linkingObjectRole valueURI="http://id.loc.gov/vocabulary/preservation/eventRelatedObjectRole/out">outcome</premis:linkingObjectRole>
+        </premis:linkingObjectIdentifier>
+
+    </premis:event>
+
+    <premis:agent>
+        <premis:agentIdentifier>
+            <premis:agentIdentifierType>VIAA SP Agent ID</premis:agentIdentifierType>
+            <premis:agentIdentifierValue>OR-319s272</premis:agentIdentifierValue>
+        </premis:agentIdentifier>
+        <premis:agentName>Rik Klein Gotink</premis:agentName>
+        <premis:agentType>person</premis:agentType>
+        <premis:agentExtension>
+            <schema:affiliation>Rik Klein Gotink fotografie</schema:affiliation>
+        </premis:agentExtension>
+    </premis:agent>
+
+</premis:premis>
+```
+
+### data/representations/representation_4/metadata/preservation/premis.xml
```

Comment:
What about the premis.xml of representation 1-3?

Reply:
I see now that you add that the premis.xml of the other representations is similar. Any reason why you pick out representation 4 as an example instead of just representation 1?